executors/cpp/coll.cpp

```diff
@@ -29,6 +29,8 @@
 #include <iostream>
 #include <string>
 
+#include "./util.h"
```

Comment:
question for here & through the other files: can we change  `"util.h"` instead of `"./util.h"`? If so, it'll appear more clearly that way.

Reply:
The simple linter recommened using "./"